pkg/tracker/listenertracker_test.go

```diff
@@ -27,10 +26,11 @@ import (
 )
 
 func TestListenerTracker(t *testing.T) {
+	t.Skip("Requires investigation since it fails on the CI environment, due to the source address is being set to nil.")
```

Comment:
Please file a follow-up before merging.

Reply:
Created the following issue to address this.
https://github.com/rancher-sandbox/rancher-desktop-agent/issues/41